pages/common/twine.md

```diff
@@ -1,21 +1,36 @@
 # twine
 
 > Utility for publishing Python packages on PyPI.
-> Subcommands such as `twine upload` have their own usage documentation.
 > More information: <https://twine.readthedocs.io>.
 
-- Upload to the Test PyPI [r]epository and verify things look right:
-
-`twine upload -r testpypi dist/*`
-
 - Upload to PyPI:
 
 `twine upload dist/*`
 
+- Upload to the Test PyPI [r]epository and verify things look right:
+
+`twine upload -r testpypi dist/*`
+
 - Upload to PyPI with a specified [u]sername and [p]assword:
 
 `twine upload -u {{username}} -p {{password}} dist/*`
 
 - Upload to an alternative repository URL:
 
 `twine upload --repository-url {{repository_url}} dist/*`
+
+- Check that your distribution’s long description should render correctly on PyPI:
+
+`twine check dist/*`
+
+- Upload with a specific pypirc configuration file (<https://packaging.python.org/en/latest/specifications/pypirc>):
```

Comment:
No links in examples.
```suggestion
- Upload using a specific pypirc configuration file:
```
It's uploaded with the pypirc inside the packages? or it's uploaded using configurations from a pypirc file?

Reply:
Uploaded using configurations from a pypirc file.